app/views/posts/index.html.erb

```diff
@@ -1 +1,43 @@
-<h2>List of posts by a given user</h2>
\ No newline at end of file
+<h1>List of all Posts by a user</h1>
+<div>
+  <div class='user_info'>
+      <img src="<%= @user.photo%>" alt="<%= @user.name%>" class='user_image'>
+      <div class='user_details'>
+        <h2>
+          <%= @user.name%>
+        </h2>
+        <h4>
+          <%= @user.posts_counter%>
+        </h4>
```

Comment:
- [ ] @habasm great work on your posts page however the number of post counter is placed wrongly and doesn't have a text indicating its the number of posts 
![image](https://github.com/habasm/BlogApp/assets/61408860/004ce543-bb04-4ff8-ad1e-171479fd5ce3)